client/api/eventsApi.js

```diff
@@ -2,6 +2,13 @@ import { getEncodedToken } from 'authenticare/client'
 import request from 'superagent'
 const acceptJsonHeader = { Accept: 'application/json' }
 
+/**
+ * Post new event to projects table
+ * @param {string} id - User Id
+ * @param {object} event - Form field inputs from CreateProject
```

Comment:
💭 We can also document `event`'s properties; check it out:
https://jsdoc.app/tags-param.html#parameters-with-properties